specification/web/resource-manager/Microsoft.Web/stable/2019-08-01/CommonDefinitions.json

```diff
@@ -1234,6 +1234,7 @@
           "enum": [
             "None",
             "SystemAssigned",
+            "SystemAssigned, UserAssigned",
```

Comment:
Add value to enum should be considered as a breaking change. So for this api-version, have you already generated the client SDK and release them to the user?

Reply:
adding a value to an enum is additive and inherently not a breaking change? by comparison removing a value would be a breaking change

Reply:
Please refer to https://github.com/Azure/adx-documentation-pr/wiki/Breaking-changes-guidelines#allowed-values-for-an-enum-have-changed

Reply:
unfortunately it appears that page is not public..

<img width="1552" alt="Screenshot 2020-02-21 at 07 16 29" src="https://user-images.githubusercontent.com/666005/75009103-15ff0800-547a-11ea-99ca-581b14005b93.png">

Reply:
Allowed values for an enum have changed 

Enum “foo” had allowed values as “val1” and “val2” in v1 of API. If now, the values allowed/accepted by the service are “val1”, “val2” and “val3”, client will fail to de-serialize if “val3” comes back in the response. For this to be non-breaking, it must be modeled as string in the swagger.

Reply:
It’s worth noting that this value was supported in previous api versions / the service, but is missing from this one - hence why this PR’s adding them back in as they can be returned from the API

Reply:
@lirenhe / @tombuildsstuff  I work at Microsoft on the Azure REST guidelines and we are in the process of formally improving these guidelines. Let me tell you where we stand on this now. Azure services can have 2 kinds of enums: regular enums and extensible enums. A regular enum is an enum with a set of values that CAN NEVER CHANGE EVER and therefore backward compatibility is always maintained - it is incumbent on the service team to ensure that the set of potential values never changes once the enum is described in Swagger.  Another way to say this is that, for a regular enum, the set of values is part of the contract and we must not break the contract even as the service versions over time - backward compatibility must always be maintained.

An extensible is enum is really just a string in Swagger and appears as "modelAsString": true. The contract is the string and while the Swagger MAY suggest values, these values are just suggested values, are NOT part of the contract, and NEW values CAN be added over time. Clients of the service should look at the string and expect that in the future, a currently-undocumented value may be returned. For extensible enums, we (Microsoft/Azure) allow values to be removed (since the values are NOT part of the contract) but we strongly discourage this. Obviously, if the service stops returning a particular value, there should be no problem in a client because it just didn't get a particular value. However, removing a value previously supported by the service CAN be problematic because the client may send it and if the service doesn't support it now, then the service may return an error while it used to return success - this would be a breaking change to a client and we'd like to avoid these as much as possible. However, we are saying to a client that, for extensible enums, our contract is that the values may come and go and so client code should be written to expect that a value that is supported today may not be supported tomorrow. The reason we allow for this is because, in some cases, we can't prevent it. For example, Azure may support some VM SKU today (represented by some value) and Azure may remove support for that VM SKU tomorrow (the hardware may just not exist in the datacenter). In this case, the service has no choice but to fail the request.